### PR TITLE
Retires Ibex and Oryx Testnets

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Removal 

**Required reviews:** 
1

### CODE REVIEW

File: nucypher/blockchain/eth/contract_registry/README.md [hunk 1]
```diff
@@ -4,24 +4,34 @@ This directory contains an authoritative source of contract registries for sever
...
+        * [**SimplePREApplication**](https://etherscan.io/address/0x7E01c9c03FD3737294dbD7630a34845B0F70E5Dd): `0x7E01c9c03FD3737294dbD7630a34845B0F70E5Dd`
+* `polygon-mainnet`: Used for cost-effective PRE subscription payments
+        * [**SubscriptionManager**](https://polygonscan.com/address/0xB0194073421192F6Cf38d72c791Be8729721A0b3): `0xB0194073421192F6Cf38d72c791Be8729721A0b3`
+* `polygon-mumbai`:
+        * [**SubscriptionManager**](https://mumbai.polygonscan.com/address/0xb9015d7B35Ce7c81ddE38eF7136Baa3B1044f313): `0xb9015d7B35Ce7c81ddE38eF7136Baa3B1044f313`
+* `tapir`: Public Long-Term Support testnet, intended as a stable playground for network users (Goerli) testnet. 
+        * [**TestnetThresholdStaking**](https://goerli.etherscan.io/address/0x09B5065d2924De33C85F76474A89A27189402064): `0x09B5065d2924De33C85F76474A89A27189402064`
+        * [**SimplePREApplication**](https://goerli.etherscan.io/address/0xaF96aa6000ec2B6CF0Fe6B505B6C33fa246967Ca): `0xaF96aa6000ec2B6CF0Fe6B505B6C33fa246967Ca`
+* `lynx`: Internal development testnet, chaotic, used for experimental code and staging. (Goerli). 
+        * TBD
```
Comment (derekpierre) [inline nucypher/blockchain/eth/contract_registry/README.md:26]: Adjust spacing for bullet points to show properly.

```suggestion
* `polygon-mainnet`: Used for cost-effective PRE subscription payments
    * [**SubscriptionManager**](https://polygonscan.com/address/0xB0194073421192F6Cf38d72c791Be8729721A0b3): `0xB0194073421192F6Cf38d72c791Be8729721A0b3`
* `polygon-mumbai`:
    * [**SubscriptionManager**](https://mumbai.polygonscan.com/address/0xb9015d7B35Ce7c81ddE38eF7136Baa3B1044f313): `0xb9015d7B35Ce7c81ddE38eF7136Baa3B1044f313`
* `tapir`: Public Long-Term Support testnet, intended as a stable playground for network users (Goerli) testnet. 
    * [**TestnetThresholdStaking**](https://goerli.etherscan.io/address/0x09B5065d2924De33C85F76474A89A27189402064): `0x09B5065d2924De33C85F76474A89A27189402064`
    * [**SimplePREApplication**](https://goerli.etherscan.io/address/0xaF96aa6000ec2B6CF0Fe6B505B6C33fa246967Ca): `0xaF96aa6000ec2B6CF0Fe6B505B6C33fa246967Ca`
* `lynx`: Internal development testnet, chaotic, used for experimental code and staging. (Goerli). 
    * TBD 
```

**Review by derekpierre — COMMENTED** [review]
Do we need to also update `NetworksInventory`?

**Comment by derekpierre** [comment]
Since this affects users who test on our testnets, should we also include a removal newsfragment?

**Comment by derekpierre** [comment]
Our docs probably also made mention of these testnets.

**Review by derekpierre — APPROVED** [review]
🎸

File: nucypher/blockchain/eth/networks.py [hunk 2]
```diff
@@ -34,8 +32,6 @@ class NetworksInventory:  # TODO: See #1564
 
     __to_chain_id_eth = {
         MAINNET: 1,  # Ethereum Mainnet
-        ORYX: 3,  # Ropsten
-        IBEX: 4,  # Rinkeby
```
Comment (cygnusv) [inline nucypher/blockchain/eth/networks.py:38]: Finally!

**Review by cygnusv — APPROVED** [review]
just a minor comment

**Pull request merged**